core/new-gui/src/app/workspace/component/property-editor/property-editor.component.ts

```diff
@@ -25,12 +29,16 @@ export type PropertyEditFrameConfig = DynamicComponentConfig<PropertyEditFrameCo
 export class PropertyEditorComponent implements OnInit {
   frameComponentConfig?: PropertyEditFrameConfig;
 
-  constructor(public workflowActionService: WorkflowActionService) {}
+  subscriptions = new Subscription();
```

Comment:
seems this `subscriptions` is not needed. we are using `UntilDestroy()` to manage the subscription life cycle now.